default.xml

```diff
@@ -12,6 +12,7 @@
   <project name="96boards/meta-96boards" path="layers/meta-96boards" remote="github" revision="master"/>
   <project name="96boards/meta-rpb" path="layers/meta-rpb" remote="github" revision="master"/>
   <project name="OSSystems/meta-browser" path="layers/meta-browser" remote="github" revision="master"/>
+  <project name="petegriffin/meta-wpe" path="layers/meta-wpe" remote="github" revision="87e8d48cdbc82b8cac5e47afc5143aca1fb10ffb"/>
```

Comment:
I believe this breaks the layer ordering.
(See e.g https://github.com/linaro-mmwg/lhg-oe-manifests/commit/7a907fb01a3e9658afaeaac284ba2a1fb4780e63)